mars/deploy/utils.py

```diff
@@ -155,6 +157,8 @@ def load_config(config: Union[str, Dict], default_config_file: str):
     else:
         full_config = load_service_config_file(default_config_file)
         config = _merge_config(full_config, config)
+    if backend:
+        config["task"]["execution_config"]["backend"] = backend
```

Comment:
Don't add the `backend` here, `backend` is for execution. You can add the `shuffle_type` to the execution config is it is related to the `backend`.

Reply:
I think it's needed.
![image](https://user-images.githubusercontent.com/12445254/168739769-3bc9fd3f-2560-4cc6-803f-c40fb29a0fd1.png)
If we don't add this, and the execution backend is `ray`, `config["task"]["execution_config"]["backend"]` will still be `mars` which is not consistent.

Reply:
Here updates the config, we can switch the backend by passing `backend="ray"` to the `new_session` and `new_test_session`.

![image](https://user-images.githubusercontent.com/6308809/168773778-5867d250-4db4-4417-8dbd-554ab3e4ca72.png)

Reply:
Got it, I passed `shuffle_type` from `TaskProcessor` to `GraphAnalyzer` directly.